lib/internal/process/promises.js

```diff
@@ -117,7 +117,8 @@ function unhandledRejection(promise, reason) {
   maybeUnhandledPromises.set(promise, {
     reason,
     uid: ++lastPromiseId,
-    warned: false
+    warned: false,
+    domain: process.domain
```

Comment:
Since this PR is adding trailing commas to `module.exports = {…}`, it should probably also add them here:

```suggestion
    domain: process.domain,
```

Reply:
We don't actually enforce (or particularly care) about trailing commas I believe?

I believe that if we do we should enforce it through eslint.